.vscode/settings.json

```diff
@@ -1,7 +1,18 @@
 {
-  "editor.insertSpaces": true,
```

Comment:
Do we need to have VSCode specific stuff in the repository? There is editorconfig: https://editorconfig.org/

Reply:
there is even a YouTrack issue for setting up editorconfig, I think. However it has not been done yet. And since we already had `.vscode` I saw no reason to not make my life easier. ;)  
But I fully agree that we should set up editorconfig to make things homogenous independent of the IDE used; and then that can definitely be removed.